WcaOnRails/app/helpers/persons_helper.rb

```diff
@@ -19,4 +19,16 @@ def odd_rank?(rank)
     any_missing = rank.continent_rank == 0 || rank.country_rank == 0 # Note: world rank is always present.
     any_missing || rank.continent_rank < rank.country_rank
   end
+
+  def return_podium_class(result)
+    if (result.roundTypeId == 'f' || result.roundTypeId == 'c') && result.best_solve.clock_format != "DNF"
```

Comment:
I'm not a fan of hardcoding the `roundTypeId` here but we don't have a good helper for testing if a result is for a final round at the moment, so it's fine as it is.
However, for DNF, we do have the `solve.dnf?` helper, so I think the last would look better using `&& !result.best_solve.dnf?`.